### PR TITLE
Onboarding improvements

### PR DESCRIPTION
**Pull request by mbianchidev**
Added env example and edited axios.js to default back to DEV instead of prod

### CODE REVIEW

File: src/axios.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const Axios = require("axios");
 const Cookies = require("js-cookie");
 
 const baseDomain =
-  process.env.NEXT_PUBLIC_BACKEND_BASE_DOMAIN || "https://api.omnistrate.cloud";
+  process.env.NEXT_PUBLIC_BACKEND_BASE_DOMAIN || "https://api.omnistrate.dev";
```
Comment (kkgupta) [inline src/axios.js:5]: @mbianchidev we can't commit. This is an external package and ".dev" is an internal endpoint. This change will break production for all of our customers.

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,4 @@
+PROVIDER_EMAIL=you@your-domain.com
+PROVIDER_PASSWORD=your-omnistrate-password
```
Comment (kkgupta) [inline .env.example:2]: Are you sure that we need this? -- I recollect @rushil95 mentioning that this is no longer needed. Can you please talk to him?

File: src/axios.js (hunk 1 above)
Comment (mbianchidev) [reply src/axios.js:5]: Shouldn't they set the env var NEXT_PUBLIC_BACKEND_BASE_DOMAIN? 
Anyway, good to know!

File: .env.example (hunk 2 above)
Comment (mbianchidev) [reply .env.example:2]: For local development I think we still need it, at least last week I had to valorize it.
We could always rename it to `.env.local` if this is the case

Comment (rushil95) [reply .env.example:2]: The customers don't use this environment variable. We use it internally for deploying to fly as we need to use the dev server for those deployments. 
@mbianchidev While running locally you could just set the baseDomain to https://api.omnistrate.dev in the above file

**Comment by mbianchidev** [comment]
Closed as it's not deemed useful in this phase by the core team

**Pull request closed without merging**